.circleci/continue_config.yml

```diff
@@ -0,0 +1,278 @@
+version: 2.1
```

Comment:
Could we make review easier by using `git mv`?

Reply:
I tried rewriting the history this way, but GitHub does still not show a useful diff. Instead, the reviewer can check out this branch and run `git diff :.circleci/continue_config.yml main:./.circleci/config.yml`, this gives a pretty clean diff.